packages/babel-helper-define-polyfill-provider/src/utils.ts

```diff
@@ -87,30 +91,33 @@ export function resolveSource(obj: NodePath) {
   return { id: null, placement: null };
 }
 
-export function getImportSource({ node }: NodePath) {
+export function getImportSource({ node }: NodePath<t.ImportDeclaration>) {
   if (node.specifiers.length === 0) return node.source.value;
 }
 
-export function getRequireSource({ node }: NodePath) {
+export function getRequireSource({ node }: NodePath<t.Statement>) {
```

Comment:
not sure if this generic is correct; this was like the least common denominator

Reply:
Yup, that's correct. It could even be `Node` since it's refined on the line after, but `Statement` is more precise.